### PR TITLE
Add docs for funding API

### PR DESCRIPTION
**Pull request by Kenadia**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1844,6 +1844,149 @@ Query: `https://api.dydx.exchange/v1/perpetual-accounts`
...
+## Funding Endpoints
+
+The funding endpoints can be used to get information related to the funding rates
+for Perpetual markets.
+
+### Funding Rate Object
+
+| Field Name                 | Description                                                                                              |
+|----------------------------|----------------------------------------------------------------------------------------------------------|
+| market                     | The market string, e.g. `PBTC-USDC`.                                                                     |
```
Comment (BrendanChou) [inline docs/api.md:1856]: unnecessary spaces before pipes `|`

File: docs/api.md [hunk 2]
```diff
@@ -1844,6 +1844,149 @@ Query: `https://api.dydx.exchange/v1/perpetual-accounts`
...
+Description:
+Get historical funding rates. The most recent funding rates are returned first.
+
+Query Params:
+
+| Field Name | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| markets    | (Optional) Markets to get rates for. Defaults to all Perpetual markets.             |
+| limit      | (Optional) The maximum number of funding rates. The default, and maximum, is 100.   |
+| offset     | (Optional) The number of entries to skip, in order to retrieve older funding rates. |
```
Comment (BrendanChou) [inline docs/api.md:1926]: change to `startingBefore`

File: docs/api.md (hunk 1 above)
Comment (Kenadia) [reply docs/api.md:1856]: Seems like just weird github formatting

**Comment by Kenadia** [comment]
Rebased with master

**Pull request merged**